hs/hie.yaml

```diff
@@ -2,6 +2,10 @@ cradle:
   stack:
     - path: "./src"
       component: "reach:lib"
+    - path: "./reach-everest/hs/src"
+      component: "reach:lib"
+    - path: "./reach-everest/hs/app-closed/reach-idd"
+      component: "reach:exe:reach-idd"
```

Comment:
If this messes up hie when you don't have Everest, we may need to put this in the Everest linking script